bucket/markdown-monster.json

```
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://markdownmonster.west-wind.com",
+    "description": "Markdown editing and weblog publishing tool.",
+    "license": "https://markdownmonster.west-wind.com/purchase.aspx#License",
```

I am not sure if this URL will change or not 🤔

```suggestion
    "license": {
        "identifier": "Proprietary",
        "url": "https://markdownmonster.west-wind.com/docs/_4s014echa.htm"
    },
```

or

```suggestion
    "license": {
        "identifier": "Proprietary",
        "url": "https://markdownmonster.west-wind.com/purchase.aspx#License"
    },
```